packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -237,11 +230,15 @@ export const DataGrid: React.FC<GridProps> = (props) => {
               : {}),
           };
         }
+        let label = null;
+        if (isString(item.label)) {
+          label = item.label;
+        } else if (isObject(item.label)) {
+          EnsembleRuntime.render([unwrapWidget({ ...item.label })]);
```

Comment:
I think 
1. we need `cloneDeep` on item.label as spread operator won't deep copy nested objects
2. assign `render` functions's output to label var

Reply:
Good catch, ty.